packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1981,9 +1984,14 @@ function updateEventComponent(current, workInProgress, renderExpirationTime) {
 }
 
 function updateEventTarget(current, workInProgress, renderExpirationTime) {
+  const type = workInProgress.type.type;
   const nextProps = workInProgress.pendingProps;
   let nextChildren = nextProps.children;
 
+  if (type === REACT_EVENT_TARGET_TOUCH_HIT && current === null) {
+    tryToClaimNextHydratableInstance(workInProgress);
+  }
+
   reconcileChildren(
```

Comment:
What does it mean to pass children to an EventTarget in this new model? I thought these would just be leafy?

Reply:
I believe hydration tests failed unless I had this – was I wrong in that thinking? EventTargets could have children, for now there is only a single EventTarget and that is a TouchHitTarget, there will be many more in the future; some that can have children, some that cannot.

Reply:
Although thinking about it. We could make them leafy, but we’d need to have special cases for void host components:

<input><FocusTarget /></input>